spec/views/components/autocomplete_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+
+RSpec.describe "Autocomplete component" do
```

Comment:
👍 For adding this test. 
Can you fix it up to the previous commit, where the component was added.
Also, there are already some component tests in `spec/components` can these be moved to the same place for consistency?